### ISSUE TITLE
Could not embed this static framework in my pod

### ISSUE THREAD

**Issue by sdc-78** [Flight-School/AnyCodable#28]
Hi,

I would like to use this library as a dependency of my own pod, but I can't, since the .podspec say it is "static". Is there a way to solve this ?

Thanx
-Sylvain

**Comment by minacle** [comment]
Add `s.static_framework = true` to your podspec.

**Comment by sdc-78** [comment]
> Add `s.static_framework = true` to your podspec.

Yes, but this would make my own pod a static library. And, I prefer to have a dynamic framework.
